_pages/en/strategy-target-solution-delivery-model.md

```diff
@@ -24,7 +24,7 @@ img + em { display: inline-block; }
 </style>
 
 <div class="alert alert-warning">
-  <strong>This is a DRAFT strategy</strong>. <a href="target-solution-delivery-model-strategy-areas-of-concerns.html">Concerns have been raised</a> by stakeholders as part of the peer review process and are being looked at.
+  <strong>This is a DRAFT strategy</strong>. It is currently in the peer review process. As <a href="target-solution-delivery-model-strategy-areas-of-concerns.html">feedbacks come in</a> the strategy will be updated to address them.
```

Comment:
Looks awesome! Only thing, and I know this makes no sense.....Feedback is never plural.  You can have a single piece of feedback of lots of feedback. But never feedbacks.  

So this would be "As feedback comes in.....".